src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -62,6 +65,7 @@ public LinuxAppServiceLogger(bool writeToConsole, string containerName, string t
             this.tenant = JToken.FromObject(tenant);
             this.sourceMoniker = JToken.FromObject(
                 string.IsNullOrEmpty(stampName) ? string.Empty : "L" + stampName.Replace("-", "").ToUpperInvariant());
+            this.procID = Process.GetCurrentProcess().Id;
```

Comment:
Does the return value of `Process.GetCurrentProcess()` need to be disposed after use?

Reply:
Yup! Thanks for the heads up

Reply:
Turns out that `Process.GetCurrentProcess()` should not be used at all for ids. https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1837.

I need to fix this in the EasyAuth codebase before releasing 1.4.0 on Linux dedicated.

Reply:
It looks like the `Environment.ProcessId` API is only available in .NET 5?

Reply:
Ah that makes sense. I thought this warning was a part of the new code warnings being added in .NET 5, but I don't see this API in .NET Standard 2.0, so the API may be new as well. In that case, just dispose of it.